src/ProjectOrigin.VerifiableEventStore/Extensions/IEnumerableMerkleExtension.cs

```diff
@@ -7,75 +7,85 @@ namespace ProjectOrigin.VerifiableEventStore.Extensions;
 
 public static class IEnumerableMerkleExtension
 {
-    public static byte[] CalculateMerkleRoot<T>(this IEnumerable<T> events, Func<T, byte[]> selector)
+    public static byte[] CalculateMerkleRoot<T>(this IList<T> events, Func<T, byte[]> selector)
```

Comment:
Why is this changed to `IList`? It looks like that it can be an `IEnumerable` and then it will match the name of the static class.

Reply:
IEnumerable interface does not guarantee order, but the order is important in this regard, therefore forcing people using the function to send a IList forces them to think about the issue.